Sources/Components/Viewings/ViewingCellViewModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct ViewingCellViewModel {
+
+    let weekday: String?
+    let month: String?
+    let date: String?
```

Comment:
Is this actually `day`/`dayOfMonth`? With name `date` it makes me think it consists of the full date

Reply:
Yeah, day makes more sense 👍